pkg/idtools/idtools_unix_test.go

```diff
@@ -251,3 +264,120 @@ dockremap:231072:65536`
 		t.Fatalf("wanted 65536, got %d instead", ranges[0].Length)
 	}
 }
+
+func TestGetRootUIDGID(t *testing.T) {
+	uidMap := []IDMap{
+		{
+			ContainerID: 0,
+			HostID:      os.Getuid(),
+			Size:        1,
+		},
+	}
+	gidMap := []IDMap{
+		{
+			ContainerID: 0,
+			HostID:      os.Getgid(),
+			Size:        1,
+		},
+	}
+
+	uid, gid, err := GetRootUIDGID(uidMap, gidMap)
+	assert.NoError(t, err)
+	assert.Equal(t, uid, os.Getegid())
+	assert.Equal(t, gid, os.Getegid())
+
+	uidMapError := []IDMap{
+		{
+			ContainerID: 1,
+			HostID:      os.Getuid(),
+			Size:        1,
+		},
+	}
+	_, _, err = GetRootUIDGID(uidMapError, gidMap)
+	assert.NotNil(t, err, "Expected error, got nil")
```

Comment:
I think this should use either `assert.ErrorEqual()` or `testutil.ErrorContains()` which will check the value is an error, and that the error message matches the expected value.